crates/viewer/re_viewer_context/src/store_hub.rs

```diff
@@ -616,9 +616,11 @@ impl StoreHub {
             return; // unreachable
         };
 
-        let store_size_before = entity_db.store().stats().total().total_size_bytes;
+        // NOTE: Do NOT try and `read_arc()` the store here: `purge_fraction_of_ram` is going to
+        // access it mutably in all sorts of ways, it's a guaranteed deadlock.
```

Comment:
That is a good example of what I mean. Thankfully that one is fairly obvious -- the hundreds that we don't know about are the real scary ones.